packages/quantic/force-app/main/default/lwc/quanticBreadcrumb/quanticBreadcrumb.css

```diff
@@ -0,0 +1,23 @@
+.align-baseline {
```

Comment:
I would suggest to follow the same styling pattern as the one from [SLDS](https://www.lightningdesignsystem.com/guidelines/markup-and-style/).

So instead of specifying the class `align-baseline` on a breadcrumb item, we could add the class `breadcrumb__item` on the `li` element that represents the breadcrumb item. Then the CSS rule would look like this:

    .breadcrumb__item {
        align-items: baseline;
    }

Following the same idea, `.text-hover-underline:hover` would change for something like this:

    .breadcrumb__item-value > a:hover {
        text-decoration: underline;
    }

Reply:
We could use BEM, I don't mind, I just viewed these as more of utility classes, but you are correct it doesn't make sense if they are defined at the component level.

I'll check if it's possible to find an already existing slds class for it, otherwise rename per your suggestion. 👍